code/modules/uplink/uplink_items.dm

```diff
@@ -2164,6 +2164,14 @@ datum/uplink_item/role_restricted/superior_honkrender
 	restricted_roles = list("Mime")
 	surplus = 0
 
+/datum/uplink_item/role_restricted/mimery/obj/item/storage/box/syndie_kit/mimesabrekit
+	name = "Baguette blade bundle"
+	desc = "A very stealthy blade located inside an even stealthier baguette-shaped sheath."
+	cost = 	12
```

Comment:
12 TC for 18 force is pretty bad, considering an energy sword is 8 TC and does considerably more damage

Reply:
What price do you suggest? This is pretty stealthy and has big armor pen. Should I up the damage?

Reply:
Upping the damage closer to an energy sword would probably be ideal, the extra 4 TC for stealthy baguette isn't a bad trade off.

Reply:
okay, got it. I will do all that in spare time since I had little time today while making it into a better state. Maybe I will remove the not seen aspect altogether, and just leave it spriteless on the player so you don't see when the mime strikes.

Reply:
hey bacon, it has 18 brute and 60 block now. so it is a higher block esword that cannot fit in pockets or backpack, but only a specialised sheath. + it counter attacks disarms

Reply:
At the time I didnt realise it inherited the armour pen from captains sabre, but whatever should be fine since its traitor only. Anyway fix line 2167
`datum/uplink_item/role_restricted/mimery/obj/item/storage/box/syndie_kit/mimesabrekit`

Reply:
Turns out 18 is not esword damage, but it is that of a spear in melee iirc

Reply:
Armour pen is very, very powerful at high armour levels.
At armour 0 18 damage is the same as 18 damage on an unarmoured target.
At armour 50 18 damage is the same as 36 damage on an unarmoured target.
At armour 80 (deathsquad) 18 damage is the same as 90 damage on an unarmoured target.
At armour 100 18 damage is the same as infinity damage on an unarmoured target.

Reply:
> At the time I didnt realise it inherited the armour pen from captains sabre, but whatever should be fine since its traitor only. Anyway fix line 2167
> `datum/uplink_item/role_restricted/mimery/obj/item/storage/box/syndie_kit/mimesabrekit`

I have fixed it, need anything more so this can get merged one day?